content/source/docs/extend/best-practices/index.html.md

```diff
@@ -9,21 +9,21 @@ description: |-
 
 # Terraform Plugin Best Practices
 
-A key feature of Terraform is it’s plugin system, separating the logic of
-managing state and configuration and providing  safe plan and apply lifecycle
+A key feature of Terraform is its plugin system, separating the logic of
+managing state, configuration, and providing safe plan and apply lifecycle
```

Comment:
This change makes it a little more confusing to read, mostly because it's hard to tell when to pop back out of the comma list. Instead, how about this: 

> A key feature of Terraform is its plugin system, which separates the details of specific vendor APIs from the shared logic for managing state, managing configuration, and providing a safe plan and apply lifecycle. Plugins are responsible for...